src/renderer/widgets/AddProxyModal/model/__tests__/submit-model.test.ts

```diff
@@ -0,0 +1,57 @@
+import { allSettled, fork } from 'effector';
+import { UnsignedTransaction } from '@substrate/txwrapper-polkadot';
+
+import { submitModel } from '../submit-model';
+import { networkModel } from '@entities/network';
+import { walletModel } from '@entities/wallet';
+import { Account, Chain } from '@shared/core';
+import { Transaction } from '@entities/transaction';
+import { SubmitStep } from '../../lib/types';
+import { initiatorWallet, testApi } from './mock';
+
+describe('widgets/AddPureProxyModal/model/confirm-model', () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  test('should fill data for confirm model for polkadot vault account', async () => {
+    const getSignedExtrinsicMock = jest.fn();
+    const submitExtrinsicMock = jest.fn();
+
+    const scope = fork({
+      values: new Map()
+        .set(networkModel.$apis, {
+          '0x00': testApi,
+        })
+        .set(walletModel.$wallets, [initiatorWallet]),
+      handlers: new Map()
+        .set(submitModel.__tests__.getSignedExtrinsicFx, getSignedExtrinsicMock)
```

Comment:
Have you tried to mock `transactionService.getSignedExtrinsic` & `transactionService.submitAndWatchExtrinsic` instead of extracting effects?